### DISCUSSION TITLE
Home Button Default Visibility

### DISCUSSION THREAD

**Discussion by amreus** [TiddlyWiki/TiddlyWiki5#6906] [Ideas]
Is there a rational for hiding the Home button by default, and not having a ui for selecting default tiddlers?

It would be more friendly show the Home button by default, and then to have a "Default Tiddler" toggle button on the View Toolbar.  Don't get me wrong, typing a list of tiddler names to show by default has its charms.  It's a small thing that would add a little shine.

**Comment by pmario** [comment]
I do support the request to have the "home button" visible a tiddlywiki.com and the empty version by default. 

The possibility to add tiddlers to the $:/DefaultTiddlers is a bit more tricky. ... I usually set my default tiddlers to "retain story ordering" which sets it to the filter `[list[$:/StoryList]]` ... and then I don't change that anymore for a long time. 

So having UI elements visible, that are used rarely is tricky. .. All of the elements in the "more" dropdown are kind of important for new users, because they don't know what's possible. .. But the more UI icons we have the more "cluttered" the UI becomes. ... So its always a balance "to show the right things"

It very much depends on the user ... Some users want a minimal UI some users want everything to be only 1 click away ...

**Comment by Jermolene** [comment]
Hi @amreus @pmario the rationale for not showing the "home" page control button by default is actually a self-imposed constraint: I try to keep the number of visible page control buttons to *three* or less (the same goes for the view and editor toolbars). Initially this was a matter of intuition: the overall visual complexity of the toolbars seemed to feel an order of magnitude higher when there are more than three. With just three items our brains seem to be able to grasp them as a group and simultaneously as separate items.

Subsequently, I have learned that this is a topic of scientific interest, first surfacing in the 1940s as the principle of [subitizing](https://en.wikipedia.org/wiki/Subitizing), which Wikipedia summarises as:

> Subitizing is the rapid, accurate, and confident judgments of [numbers](https://en.wikipedia.org/wiki/Number) performed for small numbers of items. The term was coined in 1949 by E.L. Kaufman et al.,[[1]](https://en.wikipedia.org/wiki/Subitizing#cite_note-kaufman-1) and is derived from the Latin adjective [subitus](https://en.wiktionary.org/wiki/subitus) (meaning "sudden") and captures a feeling of immediately knowing how many items lie within the visual scene, when the number of items present falls within the subitizing range.[[1]](https://en.wikipedia.org/wiki/Subitizing#cite_note-kaufman-1) Sets larger than about four items cannot be subitized unless the items appear in a pattern with which the person is familiar (such as the six dots on one face of a die). Large, familiar sets might be [counted](https://en.wikipedia.org/wiki/Counting) one-by-one (or the person might calculate the number through a rapid calculation if they can mentally group the elements into a few small sets). A person could also [estimate](https://en.wikipedia.org/wiki/Estimation) the number of a large set—a skill similar to, but different from, subitizing.

**Reply by pmario** [reply]
That's an argument.

**Comment by amreus** [comment]
I am sure we all have clicked the wrong button between only 3 choices; "Yes", "No", and "Cancel."  I do appreciate your replies, and I also agree about the importance of minimal clutter for a user interface.

I propose a system tag of `$:/tags/DefaultTiddler` to mark tiddlers default tiddlers. Then the `home` button opens these these tiddlers. A checkbox in the Info/Advanced Tab would be a good place for it. The user would immediately see the new tag appear as the toggle is clicked. It would be a good "Aha!" moment.

**Comment by pmario** [comment]
>I propose a system tag of $:/tags/DefaultTiddler to mark tiddlers default tiddlers. Then the home button opens these these tiddlers. A checkbox in the Info/Advanced Tab would be a good place for it. The user would immediately see the new tag appear as the toggle is clicked. It would be a good "Aha!" moment.

The [ControlPanel](https://tiddlywiki.com/#%24%3A%2FControlPanel) -> Info -> Basics  tab has a section: "[Default tiddlers](https://tiddlywiki.com/#%24%3A%2FDefaultTiddlers)" ... which is a filter. 

So if you add: `[tag[default]]` in there it will be enough, to tag your tiddlers that should show up as "default". .. I do use that for some of my wikis. ..

**Reply by amreus** [reply]
> : "[Default tiddlers](https://tiddlywiki.com/#%24%3A%2FDefaultTiddlers)" ... which is a filter.

Ooooh.  Yes, that will be enough.
